Replicator/ChangesFeed.cc

```diff
@@ -38,13 +38,15 @@ namespace litecore { namespace repl {
 
 
     ChangesFeed::ChangesFeed(Delegate &delegate, const Options *options,
-                             DBAccess &db, Checkpointer *checkpointer)
+                             DBAccess &db, Checkpointer *checkpointer,
+                             CollectionIndex collectionIndex)
```

Comment:
collectionIndex can be obtained by, _options->collectionSpecToIndex().find(checkpointer->getSpec())

Reply:
Hmmm not wild about it but it will work.  It's a lot of steps through several different types.

Reply:
Actually, thinking again, the map would work, because if the index is out of the range (workingCollectionCount), it would be internal logical error; checkpointer->getSpec() must have index in the range.